File: app/src/main/java/com/artmurka/artmurkaapp/model/interfacesmodel/IRequestItemList.java
package com.artmurka.artmurkaapp.model.interfacesmodel;

import com.artmurka.artmurkaapp.model.pojo.itemlist.itemlist.SuccessExample;

import io.reactivex.Observable;


public interface IRequestItemList {
    Observable<SuccessExample> getItemList(String page, String pageNumber, String sort, String order);
}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/interfacesmodel/IRequestItemList.kt
package com.artmurka.artmurkaapp.data.model.interfacesmodel

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.itemlist.SuccessExample

import io.reactivex.Observable


interface IRequestItemList {
    fun getItemList(page: String, pageNumber: String, sort: String, order: String): Observable<SuccessExample>
}
